integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/transport/test/MSMPCronForwarderCase.java

```diff
@@ -119,4 +106,39 @@ public void destroy() throws Exception {
         }
         super.cleanup();
     }
+
+    private Callable<Boolean> isServerStarted() {
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+
+                return tomcatServerManager.isRunning();
+            }
+        };
+    }
+
+    private Callable<Boolean> isLogWritten() {
+
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+
+                LogViewerClient logViewerClient =
+                        new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
+                LogEvent[] logEvents = logViewerClient.getAllSystemLogs();
+
+                boolean success = false;
+                int msgCount = 0;
+                for (int i = 0; i < logEvents.length; i++) {
+                    if (logEvents[i].getMessage().contains("Jack")) {
+                        if (4 == ++msgCount) {
```

Comment:
Shall we split this increment operator and equality check to two lines for readability?

Reply:
done